### DISCUSSION TITLE
Why wouldn't next-pwa build sw.js automatically?

### DISCUSSION THREAD

**Discussion by Elindo586** [shadowwalker/next-pwa#388] [Q&A]
Is this plugin still working? 

Say I went to install next-pwa, followed instructions, and after doing an npm run build, there was no automatic sw.js and related files created... the build comes out without any error, but just didn't create any files, no service workers, and of course no pwa page...

Is this still an active plugin?

**Comment by cksal0805** [comment]
me too.. I used next-transpile-modules with next-pwa
